src/client/styles/styles.scss

```diff
@@ -282,3 +282,30 @@ $two-thirds: $one-third * 2;
   border-color: 	#AA2A16;
   background-color: 	#AA2A16;
 }
+
+.dashboard-table {
+  font-size: 18px;
+}
+
+.dashboard-table > .govuk-table__head .govuk-table__header {
```

Comment:
Design system using BEM, so you should only need .govuk-table__header--dashboard. BEM (as much as possible) wants you to have all styles with the lowest level of specificity.